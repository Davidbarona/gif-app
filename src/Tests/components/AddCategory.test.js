import {shallow} from 'enzyme'
import { AddCategory } from '../../components/AddCategory'


describe("Testo on AddCategory Component",()=>{
    const setCategories = () =>{}
    const wrapper = shallow(<AddCategory setCategories={setCategories} />)

    test('should render AddCategory Correctly', () => {
        expect(wrapper).toMatchSnapshot()
     })

     test('must to change the text box', () => {
        const input =  wrapper.find('input') 

        const value = 'hello world'
        input.simulate('change',{target:{
            value
        }})

        expect(wrapper.find('p').text().trim()).toBe(value)
       
    })



})