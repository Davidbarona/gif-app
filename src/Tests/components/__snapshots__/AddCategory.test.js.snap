// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testo on AddCategory Component should render AddCategory Correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <p />
  <input
    input=""
    onChange={[Function]}
    type="text"
  />
</form>
`;
